backend/bin/deal-observer-backend.js

```diff
@@ -57,7 +64,7 @@ const observeActorEventsLoop = async (makeRpcRequest, pgPool) => {
       const numberOfStoredDeals = await countStoredActiveDeals(pgPool)
       if (INFLUXDB_TOKEN) {
         recordTelemetry('observed_deals_stats', point => {
-          point.intField('last_searched_epoch', newLastInsertedDeal.activated_at_epoch)
+          point.intField('last_searched_epoch', newLastInsertedDeal?.activated_at_epoch || 0)
```

Comment:
Nitpick: `||` is applied also when the left-hand side is falsy, e.g. `0` or `false`. It's usually safer to use the `??` operator instead.

```suggestion
          point.intField('last_searched_epoch', newLastInsertedDeal?.activated_at_epoch ?? 0)
```

Reply:
addressed in f73e3d8